packages/cli/src/run.js

```diff
@@ -28,7 +29,7 @@ export default async function run (_argv, hooks = {}) {
   let cmd = await getCommand(argv[0])
 
   // Matching `nuxt` or `nuxt [dir]` or `nuxt -*` for `nuxt dev` shortcut
-  if (!cmd && (!argv[0] || argv[0][0] === '-' || (argv[0] !== 'static' && fs.existsSync(argv[0])))) {
+  if (!cmd && (!argv[0] || argv[0][0] === '-' || fs.existsSync(`${argv[0]}${path.sep}pages`) || fs.existsSync(`${argv[0]}${path.sep}pages`))) {
```

Comment:
`argv[0]` is `rootDir` (while pages can be different using custom `srcDir`) but it needs a configuration then :P So we should check both with OR maybe with a utility :)

Reply:
If `nuxt.config` is missing, `rootDir` and `srcDir` is same. If we check both of `nuxt.config` and `pages` we achieve our goal, right?

Reply:
One may have `nuxt.config` but no `rootDir/pages`. So existence of either one means it is a valid nuxt directory.